src/doc/contrib/src/issues.md

```diff
@@ -225,9 +228,9 @@ There are several things to consider when triaging an issue:
     needs to discuss whether or not to proceed and what needs to be done to
     address the issue.
   * [S-needs-mentor] --- This is something the Cargo team wants to address,
-    but does not currently have the capacity to help with reviewing.
+    but does not currently have the capacity to help with reviewing. **(reserved for Cargo team)**
```

Comment:
What about `S-needs-rfc`?

> Anyone is welcome to help at this stage, but it should be clear that it is not yet accepted. However, this should only be tagged for changes that are somewhat likely to be accepted.

Should we include that or adjust its text a bit?

Reply:
I took out the "likely to be accepted".  I didn't realize that and have been tagging things that are RFC scope.  I did link out to the instructions on socializing an RFC which kind of fills a similar role of pre-vetting.